lib/Sender.js

```diff
@@ -150,7 +151,16 @@ Sender.prototype.frameAndSend = function(opcode, data, finalFragment, maskData,
     return;
   }
 
-  if (!Buffer.isBuffer(data)) {
+  // iovecs support to allow callers to avoid bufer copying for single frame
+  if (data instanceof Array) {
```

Comment:
ignore the question, found the answer -
http://stackoverflow.com/questions/22798826/in-node-js-why-is-there-a-util-isarray-and-an-array-isarray